### PR TITLE
create spell checker docker image

### PR DESCRIPTION
**Pull request by serch**
see algo https://github.com/textmaster/spell_checker/pull/1

### CODE REVIEW

File: languagetool/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
+FROM debian:stretch
+
+ENV HOME /home/languagetool
```
Comment (gottfrois) [inline languagetool/Dockerfile]: `home/app` to be consistent with all other docker images

File: languagetool/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+RUN mkdir -p $HOME
+
+ENV TMP_DIR /tmp/languagetool
+RUN mkdir -p $TMP_DIR
```
Comment (gottfrois) [inline languagetool/Dockerfile]: can we do those mkdir in the same RUN?

Comment (serch) [reply languagetool/Dockerfile]: yeah sure but why may I ask? something something docker layers I heard right? not sure

Comment (gottfrois) [reply languagetool/Dockerfile]: so everytime you add a RUN directive, it's going to create a new layer which contains all metadata from the previous one. So basically, layer size sums up and create pretty big images for nothing

**Pull request merged**